packages/tables/src/elements/Cell.tsx

```diff
@@ -5,7 +5,7 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { HTMLAttributes } from 'react';
+import React, { ThHTMLAttributes } from 'react';
```

Comment:
Should this be using `TdHTMLAttributes` as it's a table cell, not header cell?

Likewise `HeaderCell` should use `ThHTMLAttributes` like you have here.

Reply:
Doh, nice spot on both accounts.